src/main/java/dev/wakandaacademy/produdoro/usuario/domain/Usuario.java

```diff
@@ -31,14 +33,22 @@ public class Usuario {
 	private String email;
 	private ConfiguracaoUsuario configuracao;
 	@Builder.Default
-	private StatusUsuario status = StatusUsuario.FOCO;
+	private StatusUsuario status = StatusUsuario.PAUSA_CURTA;
 	@Builder.Default
 	private Integer quantidadePomodorosPausaCurta = 0;
 	
 	public Usuario(UsuarioNovoRequest usuarioNovo, ConfiguracaoPadrao configuracaoPadrao) {
 		this.idUsuario = UUID.randomUUID();
 		this.email = usuarioNovo.getEmail();
-		this.status = StatusUsuario.FOCO;
+		this.status = StatusUsuario.PAUSA_CURTA;
 		this.configuracao = new ConfiguracaoUsuario(configuracaoPadrao);
 	}
+
+
+	public void mudaStatusParaPausaCurta(UUID idUsuario) {
+		if (!this.idUsuario.equals(idUsuario)) {
+			throw APIException.build(HttpStatus.UNAUTHORIZED, "Credencial de autenticação não e válida");
+		}
```

Comment:
public void mudaStatusParaPausaCurta(UUID idUsuario) {
validaUsuario(idUsuario);
		this.status = StatusUsuario.FOCO;
	}
	
	private void validaUsuario(UUID idUsuario) {
		if (!this.idUsuario.equals(idUsuario)) {
			throw APIException.build(HttpStatus.UNAUTHORIZED, "Credencial de autenticação não e válida");
		}
	}
	
	assim tornara seu codigo mais limpo e outras pessoas conseguirão usar o seu mesmo metodo